lib/transport/test/dune

```diff
@@ -1,24 +0,0 @@
-(* -*- tuareg -*- *)
-let coverage_rewriter =
-  match Sys.getenv "BISECT_ENABLE" with
-  | "YES" -> "(preprocess (pps bisect_ppx))"
-  | _ -> ""
-| exception Not_found -> ""
-
-let () = Printf.ksprintf Jbuild_plugin.V1.send {|
-(executables
-  (names test_unit test_scale)
-  (libraries oUnit rrd-transport)
-  %s
-)
-
-(alias
-  (name runtest)
```

Comment:
Did you intend to delete the dune file for the unit test?

Reply:
It's not really deleted, I just moved it to `test/transport/dune`